tests/test_api.py

```diff
@@ -73,9 +73,13 @@ def testmentionstimeline(self):
     def testretweetsofme(self):
         self.api.retweets_of_me()
 
+    # @tape.use_cassette('testretweet.json')
     def testretweet(self):
-        # TODO(josh): Need a way to get random tweets to retweet.
-        raise SkipTest()
+    	try:
+    		tweet = self.api.user_timeline('twitter',count=1) # Get the latest tweet from @twitter
+    		self.api.retweet(tweet.id)
+    	except:
```

Comment:
This blanket exception handler renders the test useless, since if any exceptions occur, the test is skipped.